docs/makeenv.rst

```diff
@@ -40,17 +38,20 @@ It has the following behavior:
 2) if you specify the name of an environmnet that you have already downloaded, it will use this environment (NB
    currently no checks are implemented if the environment has been updated remotely, which can happen if
    we realize there were some issues with it.)
-3) if the flag `local` is set to ``False`` (default behaviour) and none of the above conditions are met, the
-   :func:`make_new` will download the data of this environment locally the first time it is called. If you don't want
-   do download anything then you can pass the flag ``local=True``
-4) if ``local=True`` (NON default behaviour) nothing will be loaded, and the :func:`make_new` will attempt to use a
+3) if you provided no arguments a default environment will be used: ``rte_case14_realistic`` 
+4) if the flag `test` is set to ``False`` (default behaviour) and none of the above conditions are met, the
+   :func:`make` will download the data of this environment locally the first time it is called. If you don't want
+   to download anything then you can pass the flag ``test=True``
+5) if ``test=True`` (NON default behaviour) nothing will be loaded, and the :func:`make` will attempt to use a
    pre defined environment provided with the python package. We want to emphasize that because the environments provided
    with this package contains only little data, they are not suitable for leaning a consistent agent / controler. That
-   is why a warning is sent in this case. Also, keep in mind that if you don't pass ``local=True`` then you will not
+   is why a warning is sent in this case. Also, keep in mind that if you don't pass ``test=True`` then you will not
    have the possibility to search for these environments provided in the package.
-5) if nothing is found, :func:`make_new` throws a EnvError.
+6) if no valid environment is found, :func:`make` throws a EnvError.
 
-TODO: explain behaviour of ".grid2opconfig.json"
+Changing cache location
+-------------------------------
+Editing the file ``~/.grid2opconfig.json`` allows you to change the data cache location. 
```

Comment:
you can do that informatically, with the function `change_local_dir`
Also maybe mention `get_current_local_dir` that get the cache, `list_available_remote_env` and `list_available_local_env` to list the environment already dowloaded.

Reply:
Improved section